Documentation/internals/cilium_operator.rst

```diff
@@ -50,6 +50,14 @@ Cilium. The following custom resources are registered by the Cilium Operator:
 -  CiliumExternalWorkload
 -  CiliumIdentity
 -  CiliumLocalRedirectPolicy
+-  CiliumEgressGatewayPolicy
+-  CiliumEndpointSlice
+-  CiliumClusterwideEnvoyConfig
+-  CiliumClusterwideEnvoyConfig
```

Comment:
Duplicate

Reply:
Actually, it should be `CiliumEnvoyConfig`. Fixed in the latest push.